tests/claim.zil.test.ts

```diff
@@ -305,6 +332,32 @@ describe("staking contract", () => {
             await increaseBNum(zilliqa, 10);
           },
           error: undefined,
+          want: {
+            events: [
+              {
+                name: "check_rewards",
+                getParams: () => ({
+                  rewards: ["List (Pair (ByStr20) (Uint128))",
+                  [
+                    [globalToken1ContractAddress,"10000000000000"],
+                    [globalToken2ContractAddress,"10000000000000"],
+                  ]
+                ]
+                }),
+              }
+            ],
+            verifyEvents: verifyEventsVerbose,
```

Comment:
```js
        verifyEvents: (events, want) => {
          for (const [index, event] of events.entries()) {
            console.log(want[index].name);
            expect(event._eventname).toBe(want[index].name);

            const wantParams = getJSONParams(want[index].getParams());
            event.params[0].value.sort((a, b) =>
              a.arguments[0] < b.arguments[0] ? 1 : -1
            );
            wantParams[0]?.value.sort((a, b) =>
              a.arguments[0] < b.arguments[0] ? 1 : -1
            );
            expect(JSON.stringify(event.params)).toBe(
              JSON.stringify(wantParams)
            );
          }
          return true;
        },
```

instead of using `verifyEventsVerbose`, can we use more simple solution for the specific cases?